homeworks/szabopeter/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,15 @@
+#adatok bekérése
+your_name = input("What's your name?")
+your_age = int(input("What's your age?"))
+python_exp_in_years = int(input("How long have you been learning Python? (Please enter a whole number, example: 10)"))
+python_developer = (input("Would you like become a Python developer? Yes or No?"))
+
+your_name = your_name.strip().title()
+python_developer = python_developer.strip().lower()
+your_age_in_days = your_age * 365
+python_developer = True if python_developer == "yes" else False
+
+if python_developer:
```

Comment:
Az if-else-ben kiprintelt szöveg a végét leszámítva ugyanaz. 
A ternary kifejezésed visszaadhatná azt a stringet ami a végén különbség a kettő között, és akkor nem is kell az if-else.
Ha mondjuk csak annyi lenne a vége hogy "You want to be a pro Python developer!" és "You don't want to be a pro Python developer!" akkor eközött a 2 között a különbség csak a "don't". Így a ternaryd visszaadhatna egy üres stringet ha pro akar lenni, és a "don't"-ot ha nem. Ezt pedig beágyazhatnád egyetlen egy print-be if-else nélkül.